db_querys_products.php

```diff
@@ -0,0 +1,57 @@
+<?php
+require 'dbConn.php';
+
+// checking that the key exists withing the array
+function check_array($key, $array){
```

Comment:
This function does not have any error handling.  What happens if the key does not exist in the array?